android/app/src/main/kotlin/com/elodin/walkie_talkie/MainActivity.kt

```
@@ -1,528 +1,540 @@
+    private var eventChannel: EventChannel? = null
+    private var controlBytesEventChannel: EventChannel? = null
+    private var bluetoothManager: BluetoothLeAudioManager? = null
+    private var audioRoutingManager: AudioRoutingManager? = null
+    private var gattServerManager: GattServerManager? = null
+    private var gattClientManager: GattClientManager? = null
+    private var hostAdvertiser: HostAdvertiser? = null
+    private var voiceTransport: L2capVoiceTransport? = null
+    private var eventSink: EventChannel.EventSink? = null
+    private var controlBytesSink: EventChannel.EventSink? = null
+
+    override fun configureFlutterEngine(flutterEngine: FlutterEngine) {
```

detekt (1.23.8)

[warning] The function configureFlutterEngine appears to be too complex based on Cyclomatic Complexity (complexity: 68). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)